const runtimeError = () => {
    // console.log(x);

    // let num = 10;
    // num();

    
}

module.exports=runtimeError;